AGENTS.md

```
@@ -1,200 +1,61 @@
+- Never run auto-formatting tools unless explicitly requested.
+- Never change config files like `eslint.config.ts` or `tsconfig.json`.
+- Never modify or patch 3rd party dependencies, including overwriting styles or using `!important`.
+- Never use `overflow: hidden`, `Array.forEach`.
+- Never disable linter checks unless explicitly requested.
+- Run `bun run ci` and `bun run build` after any code changes to ensure linting, typechecking, and build succeed.
+- Include updates to `AGENTS.md` for structural or architectural changes.
+- Use context-based cancellation in Go code.
+- Use zerolog for logging in Go.
+- Maintain plugin architecture for request kinds.
 
 ## Commands
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)